File: backup/Twitch/app/src/main/java/com/exact/twitch/ui/fragment/TopPagerFragment.java
package com.exact.twitch.ui.fragment;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import android.view.View;

import com.exact.twitch.ui.pagers.MediaPagerFragment;

public class TopPagerFragment extends MediaPagerFragment {

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        setAdapter(new TopPagerAdapter(requireActivity(), getChildFragmentManager()));
    }
}

File: app/src/main/java/com/exact/xtra/ui/pagers/TopPagerFragment.kt
package com.exact.xtra.ui.pagers

import android.os.Bundle
import android.view.View

class TopPagerFragment : MediaPagerFragment() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        setAdapter(TopPagerAdapter(requireActivity(), childFragmentManager))
    }
}
